### PR TITLE
CCM-817: Field validation tests

### PR DESCRIPTION
**Pull request by Kennedy1265**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Comment by PhilipSkinner** [comment]
Nice, good job catching these test cases @Kennedy1265

File: tests/test_field_validation.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+NHS Number must be 10 digits long
+DOB must be in the format of YYYY-MM-DD
+"""
+import requests
+import pytest
+
+
+NHS_NUMBER = ["012345678", "01234567890", "abcdefghij"]
+DOB = ["1990-10-1", "1990-1-10", "90-10-10", "10-12-1990", "1-MAY-2000", "1990/01/01"]
+
```
Comment (PhilipSkinner) [inline tests/test_field_validation.py:13]: You could expand on this and add tests for the uuid validation on:

* routingPlanId
* messageBatchReference
* messageReference

**Review by PhilipSkinner — CHANGES_REQUESTED** [review]
Please add tests for the missing test cases.

File: tests/test_field_validation.py (hunk 1 above)
Comment (Kennedy1265) [reply tests/test_field_validation.py:13]: Good shout! I'll add those in

Comment (PhilipSkinner) [reply tests/test_field_validation.py:14]: the new tests look good, all passing on the CI, resolving.

**Review by PhilipSkinner — APPROVED** [review]
Looks good, approving.

**Pull request merged**